# -*- coding: utf-8 -*-
# Generated by Django 1.11.6 on 2017-10-25 22:18
from __future__ import unicode_literals

import django.contrib.auth.validators
from django.db import migrations, models
import member.custom_validators


class Migration(migrations.Migration):

    dependencies = [
        ('member', '0009_auto_20171025_2146'),
    ]

    operations = [
        migrations.AlterField(
            model_name='user',
            name='age',
            field=models.IntegerField(blank=True, null=True, verbose_name='나이'),
        ),
        migrations.AlterField(
            model_name='user',
            name='username',
            field=models.CharField(error_messages={'unique': 'A user with that username already exists.'}, max_length=150, unique=True, validators=[django.contrib.auth.validators.UnicodeUsernameValidator(), member.custom_validators.validate_fb_username], verbose_name='username'),
        ),
    ]
